index.d.ts

```diff
@@ -88,6 +90,46 @@ export interface Options {
 	```
 	*/
 	readonly maximumFractionDigits?: number;
+
+	/**
+	The number of spaces to put between the number and unit.
+
+	If it is not set, the default value is to display 1 space between the number and unit.
+
+	 If it is set to a negative number, the number of spaces between the number and unit will be 0.
+
+	@default 1
+
+	@example
+	```
+	import prettyBytes from 'pretty-bytes';
+	prettyBytes(1920, {spaces: 0});
+	//=> '1.9kB'
+
+	prettyBytes(1920);
+	//=> '1.92 kB'
+	```
+	*/
+	readonly spaces?: number;
```

Comment:
I don't see the point in allowing multiple spaces. Make it `space: boolean`.

Reply:
I agree. In this case, I'd create an option like `noSpace`, so that the default value for it can be `false` when not supplied by the user - or are you okay with `space`'s default value being `true`?

Reply:
It should be "space"